src/constants_32bit.rs

```diff
@@ -75,6 +75,9 @@ pub const HALF: FieldElement32 = FieldElement32([
 pub const A: FieldElement32       = FieldElement32([
     486662, 0, 0, 0, 0, 0, 0, 0, 0, 0, ]);
 
+/// `APLUS2_OVER_FOUR` is (A+2)/4. (This is used internally within Montgomery laddering.)
```

Comment:
nitpick: "within the Montgomery ladder"?

Reply:
What's weird about saying that?

Reply:
Oh, I see, I thought my comment was saying "within the Montgomery ladder" and you were saying that was weird, but instead I'm *not* saying that and it *is* in fact weird.